homework18/src/ba/bitcamp/homework18/task01/AddComplain.java

```diff
@@ -0,0 +1,110 @@
+package ba.bitcamp.homework18.task01;
+
+import java.awt.Color;
+import java.awt.FlowLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Calendar;
+
+import javax.swing.BorderFactory;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+
+public class AddComplain extends JFrame {
+	
+	private static final long serialVersionUID = 7691213307856276530L;
+	private JPanel panel = new JPanel();
+	private JTextField nameField = new JTextField(20);
+	private JTextArea areaField = new JTextArea(10,20);
+	private JLabel nameLabel = new JLabel("Your name:");
+	private JLabel areaLabel = new JLabel("Your complain:");
+	private JButton send = new JButton("Send complain");
+	private JButton view = new JButton("View complains");
+	
+	SQLConn connection = null;
+	Statement statement = null;
+	
+	/*
+	 * Constructor of GUI for adding complains.
+	 * There is initialized statement for creating table in database (if not exists),
+	 * and there are two buttons with ActionListeners.
+	 */
+	public AddComplain(SQLConn connection) {
+		this.connection = connection;
+		
+		try {
+			statement = connection.getConn().createStatement();
+			String createTable = "create table IF NOT EXISTS complain (id Integer primary key, name varchar(20), text varchar(255), date varchar(255), time varchar(255))";
```

Comment:
Vidim da svi nesto generalno imate problem sa datumom u tabelama :)?
Znaci trebao si prilikom pravljenja tabele postaviti da su kolone date i time tipa Date.
Takodjer zasto imas date i time (ako je odgovor zbog vremenskog dijela u datumu onda se nemoras sikirati jer baza to cuva bez problema. Samo je bitno sa kojim Java tipom citas i pises u bazu da bi imao vrijeme, a to je Java Timestamp.